### PR TITLE
Edit contact feature file added

### PR DESCRIPTION
**Pull request by Samikshap85**
### description
This PR adds the Edit contact feature  

### Issues
https://github.com/SagarGi/FeatureFiles/issues/6

### CODE REVIEW

File: Features1/tests/features/editContact.feature [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    I want to edit my existing contacts 
+    So that i can make some update and display it in my contact list
+
+    Background:
+        Given a user is navigated to the contact page 
+        And has following contact list 
+            |name|email        |phone   |contact-type|
+            |abc |abc@gmail.com|98000000|Personal    |
+            |aaa |aaa@gmail.com|98111111|Professional|
+        And clicks on the "Edit" button , data is displayed on the edit contact form
```
Comment (SagarGi) [inline Features1/tests/features/editContact.feature]: you have created 2 users so 2 edit button is there which edit button would you click?

**Comment by SagarGi** [comment]
@Samikshap85  rather thank making editing each scenario make a scenario that creates a single user and edit its information and like wise make multiple user and edit specific user data. it would be enough

File: Features1/tests/features/editContact.feature [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        
+
+        
+        Scenario: Edit name only
+            When  the user enters name as "samiksha" 
+            And clicks on updates contact button
+            Then  the contact name should be updated
```
Comment (kiranadh1452) [inline Features1/tests/features/editContact.feature]:
```suggestion
            Then  the contact name should be updated to "samiksha"
```

File: Features1/tests/features/editContact.feature [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        Scenario: Edit email only
+            When  the user edits email as "samiksha@abc.com"
+            And clicks on update contact button
+            Then  the contact email should be updated
+
+        
+        Scenario: Edit phone only
+            When  the user edits phone as "98001000"
+            And clicks on update contact button
+            Then  the contact phone should be updated
```
Comment (kiranadh1452) [inline Features1/tests/features/editContact.feature]:
```suggestion
            Then  the contact phone should be updated to "98001000"
```

File: Features1/tests/features/editContact.feature [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+        
+        Scenario: Edit contact type only
+            When  the user edits contact type as "Personal"
+            And clicks on update contact button
+            Then  the contact type should be updated
```
Comment (kiranadh1452) [inline Features1/tests/features/editContact.feature]:
```suggestion
            Then  the contact type should be updated to "Personal"
```

File: Features1/tests/features/editContact.feature [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+
+        Scenario: Edit all inputs fields
+            When the user edits all the input fields from following
+                |name     |email                 |phone   |
+                |samikshya|samiksha2346@gmail.com|98723144|
+            Then the contact should be updated with the new data
```
Comment (kiranadh1452) [inline Features1/tests/features/editContact.feature]: Check if the new data is actually updated.

File: Features1/tests/features/editContact.feature [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+
+        
+
+        
+    
+
+
+
+
```
Comment (kiranadh1452) [inline Features1/tests/features/editContact.feature]:
```suggestion
```

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

**Review by kiranadh1452 — APPROVED** [review]

**Pull request merged**